tests/unit/test_common.py

```diff
@@ -0,0 +1,89 @@
+import pytest
+
+from entitysdk import common as test_module
+from entitysdk.exception import EntitySDKError
+from entitysdk.typedef import DeploymentEnvironment
+
+VLAB_ID = "ff888f05-f314-4702-8a92-b86f754270bb"
+PROJ_ID = "f373e771-3a2f-4f45-ab59-0955efd7b1f4"
+
+
+@pytest.mark.parametrize(
+    ("url", "expected"),
+    [
+        (
+            f"https://staging.openbraininstitute.org/app/virtual-lab/lab/{VLAB_ID}/project/{PROJ_ID}/home",
+            test_module.ProjectContext(
+                virtual_lab_id=VLAB_ID,
+                project_id=PROJ_ID,
+                environment=DeploymentEnvironment.staging,
+            ),
+        ),
+        (
+            f"https://staging.openbraininstitute.org/app/virtual-lab/lab/{VLAB_ID}/project/{PROJ_ID}",
+            test_module.ProjectContext(
+                virtual_lab_id=VLAB_ID,
+                project_id=PROJ_ID,
+                environment=DeploymentEnvironment.staging,
+            ),
+        ),
+        (
+            f"https://openbraininstitute.org/app/virtual-lab/lab/{VLAB_ID}/project/{PROJ_ID}/home",
```

Comment:
At the moment the production domain is `www.openbraininstitute.org`, not `openbraininstitute.org`

Reply:
Good catch. Fixed it.